Index: OpenRTM-aist/src/lib/rtm/FsmActionListener.h
===================================================================
--- OpenRTM-aist/src/lib/rtm/FsmActionListener.h	(リビジョン 2806)
+++ OpenRTM-aist/src/lib/rtm/FsmActionListener.h	(作業コピー)
@@ -29,9 +29,7 @@
 
 namespace RTC
 {
-  struct FsmStructure
-  {
-  };
+  struct FsmStructure;
   /*!
    * @if jp
    *
Index: OpenRTM-aist/src/lib/rtm/SdoServiceAdmin.cpp
===================================================================
--- OpenRTM-aist/src/lib/rtm/SdoServiceAdmin.cpp	(リビジョン 2806)
+++ OpenRTM-aist/src/lib/rtm/SdoServiceAdmin.cpp	(作業コピー)
@@ -82,6 +82,7 @@
     
     // If types include '[Aa][Ll][Ll]', all types enabled in this RTC
     ::coil::vstring activeProviderTypes;
+/*
     for (size_t i(0); i < enabledProviderTypes.size(); ++i)
       {
         std::string tmp(enabledProviderTypes[i]);
@@ -100,6 +101,8 @@
               }
           }
       }
+*/
+    activeProviderTypes = availableProviderTypes;
 
     SdoServiceProviderFactory& factory(SdoServiceProviderFactory::instance());
     for (size_t i(0); i < activeProviderTypes.size(); ++i)
